homeassistant/helpers/config_validation.py

```diff
@@ -1026,6 +1027,13 @@ def STATE_CONDITION_SCHEMA(value: Any) -> dict:  # pylint: disable=invalid-name
     has_at_least_one_key("before", "after", "weekday"),
 )
 
+TRIGGER_CONDITION_SCHEMA = vol.Schema(
+    {
+        **CONDITION_BASE_SCHEMA,
+        vol.Required("trigger_id"): str,
```

Comment:
Would be nice if it could accept one or more trigger IDs.

e.g.,

```yaml
- condition: trigger
  trigger_id: my_id
```

```yaml
- condition: trigger
  trigger_id:
    - some_id
    - some_other_one
```

Reply:
Fixed in bf2e071